app/assets/social.json

```diff
@@ -1,5 +1,6 @@
 {
-	"email_template": "Salam sejahtera,\n\n#{text}\n\nSalam,\nTim KTO Matematika",
+	"email_template": "Salam sejahtera,\n\n#{text}\n\nSalam,\nTim KTO Matematika.",
+	"email_template_notif": "Salam sejahtera,\n\n#{text}\n\nSalam,\nTim KTO Matematika\n\nAnda menerima email ini karena Anda subscribe ke notifikasi kontes kami. Klik link ini untuk berhenti menerima email dari kami: #{url}",
```

Comment:
Let's use `notification_email_template` or `email_template_with_unsubscribe_link` as keys?

Try to not shorten names in code.

Reply:
I choose email_template_with_unsubscribe_link